### ISSUE TITLE
Possibility to add feature to project

### ISSUE THREAD

**Issue by sirex** [python-dirbtuves/website#2]
Only registered users should be able to add features to projects.

You need to implement a form to add feature and list of features displayed below project description. I think you can create a separate app for this thing.

**Comment by sirex** [comment]
I think it would be great to use Github API for this. All project features would be Github issues with "backlog" label. And then, everyone could vote for those features.

**Comment by niekas** [comment]
Agreed. I also think that it would be great to have tree structure (e.g. mptt) to organise features. It would be easier to track dependencies between tasks.
